priv/repo/migrations/20160802030642_create_note.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Notox.Repo.Migrations.CreateNote do
+  use Ecto.Migration
+
+  def change do
+    create table(:notes) do
+      add :title, :string
+      add :description, :string
+
+      timestamps
```

Comment:
And here. Note, that while this may look like a different syntax, it's just normal functions that have dropped the parenthesis, so this one would be `timestamps null: false`